rotkehlchen/chain/evm/node_inquirer.py

```diff
@@ -536,9 +536,12 @@ def _query(self, method: Callable, call_order: Sequence[WeightedNode], **kwargs:
             return result
 
         # no node in the call order list was succesfully queried
-        raise RemoteError(
+        log.error(
             f'Failed to query {method!s} after trying the following '
-            f'nodes: {[str(x) for x in call_order]}. Check logs for details.',
+            f'nodes: {[x.node_info.name for x in call_order]}',
+        )
+        raise RemoteError(
+            f'Please check your network and confirm sufficient nodes are connected for the blockchain to function correctly.'
```

Comment:
hey this is not what we meant. We meant to show the actual chain. Yabir explained how you can do it.
```suggestion
            f'Please check your network and confirm sufficient nodes are connected for {str(self.blockchain)}'
```

Reply:
Resolved!
@LefterisJP